webapp.py

```diff
@@ -47,6 +51,16 @@ def predict_from_webcam():
 
   prediction_name, confidence = get_yolo_predictions(yolo_model, img, 0)
 
+  if caution_switch == True:
+    if prediction_name in caution_items:
+      tts = gTTs('caution')
+      tts.save('caution.mp3')
+      playsound.playsound('caution.mp3')
+
+
+
+
+
   return jsonify(f'{prediction_name} with confidence of {confidence} percent')
```

Comment:
I'd remove most of all the code above since mp3 files won't be as helpful on the server.  

Instead I would just modify the JSON string returned to the client if the item is a caution item. 

Something like

```
if prediction_name in caution_items:
  return jsonify(f'Caution! {prediction_name} with confidence of {confidence} percent')
else: 
  return jsonify(f'{prediction_name} with confidence of {confidence} percent')